ru/turing/courses/lesson3/Yemelyanov/VodkaContainedType.java

```diff
@@ -0,0 +1,10 @@
+package ru.turing.courses.lesson3.Yemelyanov;
+
+public class VodkaContainedType extends Alcohol{
```

Comment:
```suggestion
public class Vodka extends Alcohol{
```
Странное название для класса